test/reducers/datepicker.js

```diff
@@ -16,16 +16,16 @@ test('verify initial state', (t) => {
   t.deepEqual(actual, inputstate);
 });
 
-test('reducer sets initial state to "Recent" on first initialization run.', (t) => {
-  const testStart = Date.now();
-  sinon.useFakeTimers(testStart);
+test('reducer sets initial state to "Custom" on first initialization run.', (t) => {
```

Comment:
I don't think this is true anymore.  It should just determine the label from the props I think.(?)

Reply:
Yeah I think I worded this wrong for what I'm trying to test, which is just the initial state. Changed the title and cleaned it up.